### PR TITLE
DAOS-10339 test: Fix inconsitant size test

### PR DESCRIPTION
**Pull request by knard-intel**
# Description

The SCM size of a pool created could eventually be greater than the one provided to dmg.  Thus, the size given to the dmg is not reliable and should not be used for consistancy checks.

Quick-Functional: true
Test-tag: list_verbose_basic
Test-repeat: 20
Required-githooks: true

Signed-off-by: Cedric Koch-Hofer <cedric.koch-hofer@intel.com>

### CODE REVIEW

**Comment by knard-intel** [comment]
Restart CI test as some impacted tests have not been launched: too restrictive tags.

**Review by shimizukko — APPROVED** [review]

**Comment by knard-intel** [comment]
All the tests impacted by this PR have been run 20 times.

File: src/tests/ftest/pool/list_verbose.py [hunk 1]
```diff
@@ -177,7 +177,7 @@ def verify_pool_lists(self, targets_disabled, scm_size, nvme_size):
             # Verify scm_size using the threshold rather than the exact match.
             rank_count = len(pool.target_list.value)
             if scm_size[index] is None:
-                created = pool.scm_size.value * rank_count
+                created = pool.scm_per_rank * rank_count
```
Comment (daltonbohning) [inline src/tests/ftest/pool/list_verbose.py:180]: This seems to ultimately compare scm size from `pool create` and `pool list`. Is that the expected behavior of this test?

Comment (knard-intel) [reply src/tests/ftest/pool/list_verbose.py:180]: Yes, from what I understand. The issue there was to compare the size given to the command line and not the real SCM size of the created pool.
For the history, initially the dmg command was not returning this last size, but the size given to the command line. Thus, previously to the landing of the pull request [8864](https://github.com/daos-stack/daos/pull/8864), using pool.scm_size.value or pool.scm_per_rank was the same.
Please, @shimizukko could you confirm or rectify my statements.

**Review by daltonbohning — APPROVED** [review]

File: src/tests/ftest/pool/list_verbose.py (hunk 1 above)
Comment (daltonbohning) [reply src/tests/ftest/pool/list_verbose.py:180]: Fine with me. Thanks for the explanation!

**Pull request merged**